app/web/features/messages/requests/HostRequestSendField.tsx

```diff
@@ -131,6 +133,42 @@ export default function HostRequestSendField({
 
   return (
     <form onSubmit={onSubmit}>
+      {isHost &&
+        hostRequest.status ===
+          HostRequestStatus.HOST_REQUEST_STATUS_PENDING && (
```

Comment:
What about making this a a variable `isPendingHost` and reference that instead for readability?

Reply:
Done!